cloudtrail-lambda-dynamo-cdk/README.md

```diff
@@ -29,6 +28,10 @@ s.
     ```
     npm install
     ```
+1. Open the lambda function cloudtrail-lambda-dynamo-cdk/src/lib/lambda/object_tag_checker/index.py, and on line 17 replace the example keys with your required keys:
+    ```
+    required_keys = {"Key1", "Key2", "Key3", "Key4"}
+    ```
```

Comment:
Can you please explain what these resource keys are. Are they S3 object keys?

Reply:
Yes they're S3 object keys, added this in the latest commit as well